nordeck/matrix-widget-toolkit
# Fix the package.json exports and actually provide the typescript types

To test this you can do this:

1. Run `yarn build`
2. Go into a package
3. Run `node ../../scripts/prepack.js`
4. Run `npx publint`
5. Verify its happy





**:heavy_check_mark: Checklist**



- [ ] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).
